packages/components/autocomplete/src/autocomplete.vue

```diff
@@ -151,6 +154,7 @@ const loading = ref(false)
 
 const listboxId = computed(() => ns.b(String(generateId())))
 const styles = computed(() => rawAttrs.style as StyleValue)
+const isFitInputWidth = computed(() => props.fitInputWidth)
```

Comment:
Why not use it directly?

Reply:
You are right, I just copied form el-select. :D